.cirrus.yml

```diff
@@ -88,15 +88,17 @@ osx_task:
     depends_on:
         - validate
     macos_instance:
-        image: catalina-xcode
+        image: ghcr.io/cirruslabs/macos-ventura-base:latest
     setup_script: |
         # /usr/local/opt/go@1.16 will be populated by (brew install go@1.16) below
-        export PATH=$GOPATH/bin:/usr/local/opt/go@1.16/bin:$PATH
+        # /opt/homebrew is the expected path for anything installed with brew
+        # on M1 macs
+        export PATH=$GOPATH/bin:/usr/local/opt/go@1.16/bin:/opt/homebrew/opt/go@1.16/bin:/opt/homebrew/bin:$PATH
```

Comment:
Per the other link, `/usr/local/` could possibly contain Intel binaries, and `/opt/homebrew` ARM ones, so `/opt/homebrew` should be preferred. (That might well not make any difference in practice.)

We don’t expect to go back to non-M1, so the `/usr/local/…` paths can maybe be dropped entirely.

(Perhaps modify the script for debugging purposes to `ls -la /usr/local/opt/go*/bin/* /opt/homebrew/opt/go*/bin/* /opt/homebrew/bin `; then we would see what we are dealing with in the failure logs.)